CoreQualityControlView/test/unit/src/au/gov/asd/tac/constellation/views/qualitycontrol/widget/DefaultQualityControlAutoButtonNGTest.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
```

Comment:
@Auriga2 please update the code templates following https://github.com/constellation-app/constellation/blob/master/CONTRIBUTING.md#pull-requests